build.sbt

```diff
@@ -164,9 +164,9 @@ lazy val docs = project
       "version"                 -> version.value,
       "scaladoc.skunk.base_url" -> s"https://static.javadoc.io/org.tpolecat/skunk-core_2.12/${version.value}",
       "scaladoc.fs2.io.base_url"-> s"https://static.javadoc.io/co.fs2/fs2-io_2.12/${fs2Version}",
+      "github.base_url"         -> s"https://github.com/tpolecat/skunk/tree/v${version.value}"
     ),
     mdocIn := (baseDirectory.value) / "src" / "main" / "paradox",
-    Compile / paradox / sourceDirectory := mdocOut.value,
```

Comment:
Hm, it's important for this to be here, otherwise code samples aren't evaluated. Am I missing something?

Reply:
Apologies for not getting back sooner.  To be honest, I am not super comfortable with SBT and didn't thoroughly check the docs to make sure code samples were still included.  I thought it would be a simple fix, but guess not ☹️ .   I tried to play with this for a bit more and just couldn't seem to make the codegen and source path resolve correctly together with SBT.  I may try some other time but will close the PR for now.